### PR TITLE
Postgres PLEASE DON'T MERGE YET

### PR DESCRIPTION
**Pull request by mchung1128**

### CODE REVIEW

File: server/db.js [hunk 1]
```diff
@@ -1,23 +1,56 @@
...
+  review: Sequelize.JSON,
+  popularity: Sequelize.INTEGER
+}, { sequelize, modelName: 'review' });
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connected to db!')
+  })
+  .catch(err => {
```
Comment (willresen) [inline server/db.js]: This `.catch()` isn't actually handling your error - the error will be consoled automatically. If you aren't going to handle an error, you don't need `.catch()`!

File: server/index.js [hunk 2]
```diff
@@ -13,29 +13,75 @@ app.use(cors());
 
 app.use(express.static('public'));
 
+// app.get('/reviews/:id', (req, res) => {
```
Comment (willresen) [inline server/index.js]: You may want to consider creating another file instead of commenting out the legacy endpoints.

File: server/db.js (hunk 1 above)
Comment (mchung1128) [reply server/db.js]: Took it out.

File: server/index.js (hunk 2 above)
Comment (mchung1128) [reply server/index.js]: Good call, moved it to another file and adjusted the necessary paths in my seed.

**Pull request merged**